production/sample/default/main.jsonnet

```diff
@@ -7,8 +7,8 @@ prometheus + promtail + {
   // from Loki logs to Jaeger traces on traceID.
   local service = $.core.v1.service,
   _images+:: {
-    grafana: 'grafana/grafana:7.3.x-exemplars',
-    prometheus: 'cstyan/prometheus:exemplars-64206a',
+    grafana: 'grafana/grafana:latest',
+    prometheus: 'prom/prometheus:latest',
```

Comment:
Lets not pin "latest" - thats how the demo breaks in the future.  Update this to a reasonable version.

Reply:
(Perhaps even try and keep it in sync with the docker-compose?)